README.md

```diff
@@ -1,12 +1,14 @@
 # Music Releases
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project we practiced React and created components by using data from Spotify. We build a page which shows new album and single releases. 
+This project was done by Maria Thomasson and Birgit Nehrwein.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We build the site using three components (Album.js, Artist.js and Header.js) in React. It was a challenge to understand where to add what in the different files that we created.
+We used ressources from Stack Overflow, Slack and Github for our project.
+If we had more time, we would add a list of playlists in a sidebar.
+
```

Comment:
We saw your comment that you would like to know more how the components interact with App.js and how the data is injected into props. 
This is how we understand it:
*Props are arguments passed into React components. React use HTML attributes to pass props. 
So props are like function arguments in JavaScript and attributes in HTML. It feels a bit unusual to pass data thought attributes, but I guess we have to get use to it. 
*Components are like functions that return HTML elements. Components are reusable bits of code, you can use one Component several times. For a visual understanding:
https://www.techdiagonal.com/wp-content/uploads/2019/09/react-props-blog-image-design.jpg